src/core/utils/index.js

```diff
@@ -314,3 +314,14 @@ export const addQueryParamsToHistory = ({
 }) => {
   return qhistory(history, _stringify, _parse);
 };
+
+export const getPreviewImage = (addon, { version = 0, full = true } = {}) => {
```

Comment:
Can we move this function to `src/core/imageUtils.js`?

Reply:
Is it really a `version` or just an index?

Reply:
yeh index is better, thanks

Reply:
yep, I'll move that too..